convince-my-boss.html

```diff
@@ -36,35 +36,35 @@
   <!-- <div id="preloader">
     <div id="load"></div>
   </div> -->
-  <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
-    <div class="container">
-      <img src="img/banner-logo.png" class="logo" alt="banner-logo" id="logo" style="width: 150px;">
-      <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"
-            aria-expanded="false">
-          <span class="sr-only">Toggle navigation</span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-        </button>
-      </div>
+    <nav class="navbar navbar-custom navbar-fixed-top top-nav-collapse" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
+      <div class="container">
+        <a class="navbar-brand" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
+          <img src="./img/banner-logo.png" class="logo" alt="banner-logo" id="logo" style="width: 150px; display: inline-block;">
+        </a>
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+        </div>
 
-      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-        <ul class="nav nav-justified">
-          <li><a href="index.html#home">Home</a></li>
-          <li><a href="index.html#journey">Journey</a></li>
-          <li><a href="index.html#speakers">Speakers</a></li>
-          <!-- <li><a href="index.html">Schedule</a></li> -->
-          <li><a href="index.html#sponsors">Sponsors</a></li>
-          <li><a href="index.html#tickets">Tickets</a></li>
-          <li><a href="https://in.pycon.org/blog/">Blog</a></li>
-          <li><a href="index.html#venue">Venue</a></li>
-          <li><a href="faq.html">FAQ</a></li>
-          <li><a href="coc.html">COC</a></li>
-        </ul>
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```

Comment:
@dhavalsavalia please remove the extra tabs in lines from 52 - 64. These are all unnecessary changes